app/src/main/java/com/example/pokedexdesafio/domain/repository/GetPokemonDetailsRepo.kt

```diff
@@ -1,10 +1,10 @@
 package com.example.pokedexdesafio.domain.repository
 
-import com.example.pokedexdesafio.data.model.PokemonDetail
+import com.example.pokedexdesafio.data.model.PokemonDetailResponse
 import io.reactivex.rxjava3.core.Single
 import retrofit2.Response
 
 interface GetPokemonDetailsRepo {
 
-    fun getPokemonDetails(id: String): Single<Response<PokemonDetail>>
+    fun getPokemonDetails(id: String): Single<Response<PokemonDetailResponse>>
```

Comment:
Thi s is not expected. You are still using a model from data instead a model from business